api/v1beta1/verticadb_webhook.go

```diff
@@ -869,6 +876,26 @@ func (v *VerticaDB) validateLocalPaths(allErrs field.ErrorList) field.ErrorList
 			fieldRef, fmt.Sprintf("%s cannot be set to %s. This is a restricted path.", fieldPathName, invalidPath))
 		allErrs = append(allErrs, err)
 	}
+	// When depotVolume is EmptyDir, depotPath must be different from data and catalog paths.
+	if v.IsDepotVolumeEmptyDir() {
+		if v.Spec.Local.DepotPath == v.Spec.Local.DataPath ||
```

Comment:
Now I see what you mean and this webhook rule will reject the update on this specific scenario. A way to fix this would be to also update depotVolume to `PersistentVolume` and log a msg like `cannot use emptyDir as depot volume to revive db with depotPath=dataPath, a persistent volume will be used.`